### ISSUE TITLE
[scheduler] GitHub checks are not being updated correctly

### ISSUE THREAD

**Issue by CaseyHillers** [flutter/flutter#84450]
No pubsub data was sent for 7735 - https://ci.chromium.org/ui/p/flutter/builders/try/Mac%20module_test/7735/overview (see https://github.com/flutter/flutter/pull/84408)

This caused the engine roll to be stuck on a pending build. We should create a cron vacuum that will manually checking pending tests and verify their status with LUCI.

**Comment by CaseyHillers** [comment]
\cc @zanderso

**Comment by godofredoc** [comment]
When this happen, let's take a look at the # of messages in the pubsub queue.

**Comment by CaseyHillers** [comment]
I don't this was accurate. After more investigation last week, it appears that the GitHub API isn't actually writing the result, or the pubsub results come out of order.

**Comment by godofredoc** [comment]
Closing because the bug has not been updated for more than one year.

Feel free to reopen any bugs that you believe are important but please provide updates on how to resource the fix of the bug and what can be done to prevent the issue from not getting updates for another year.

**Comment by zanderso** [comment]
@CaseyHillers Was the work you outlined in https://github.com/flutter/flutter/issues/84450#issuecomment-1077866564 completed?

**Comment by CaseyHillers** [comment]
> @CaseyHillers Was the work you outlined in https://github.com/flutter/flutter/issues/84450#issuecomment-1077866564 completed?

No. This bug should still be open as Cocoon won't recover from outages correctly
